packages/netlify-cms-core/index.d.ts

```diff
@@ -23,7 +23,20 @@ declare module 'netlify-cms-core' {
     | 'bulleted-list'
     | 'numbered-list';
 
-  export type CmsFilesExtension = 'yml' | 'yaml' | 'toml' | 'json' | 'md' | 'markdown' | 'html';
+  export interface CmsSelectWidgetOptionObject {
+    label: string;
+    value: any;
+  }
+
+  export type CmsFilesExtension =
+    | 'yml'
+    | 'yaml'
+    | 'toml'
+    | 'json'
+    | 'md'
+    | 'markdown'
+    | 'html'
+    | string;
```

Comment:
```suggestion
  export type CmsFilesExtension = string;
```
I think we can safely change this to a `string`

Reply:
Thanks, @erezrokah! I've updated it.